recipes/macfsevents/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "macfsevents" %}
+{% set version = "0.7" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "95d3cddaf8a42435bfbd50087785ee9e3ebb8325242cfd06a88ea21f85bcb56f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.python.org/packages/f4/fb/59f72719e339f6209997414c6d0b7e1e1f96900dede3ec1cc24008471cc2/MacFSEvents-0.7.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [not osx]
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - fsevents
+
+about:
+  home: https://github.com/malthe/macfsevents
+  license: MIT
+  license_family: MIT
```

Comment:
Repo says BSD

Reply:
Yeah, Pypi metadata is inconsistent with the README. I've asked the upstream author [to clarify](https://github.com/malthe/macfsevents/issues/38)